CMakeLists.txt

```diff
@@ -56,6 +56,8 @@ option(PLLGPU_ENABLE_COVERAGE "Enable code coverage" OFF)
 option(PLLGPU_ENABLE_OPENMP "Enable OpenMP support" ON)
 option(PLLGPU_ENABLE_MPI "Enable MPI support" OFF)
 
+set(MPICH OFF)
```

Comment:
Since these two variables (MPICH and SYTEM_NAME) are just for Permulter with MPICH. I found an env variable (CRAY_MPICH_VERSION) is set once Cray-MPICH is loaded. We can discuss on it later.